### PR TITLE
Apply pending rewards to stats

### PR DESCRIPTION
**Pull request by peroxy**
Apply pending rewards to metrics/loans/owners endpoints

### CODE REVIEW

**Review by y2kappa — APPROVED** [review]
needs one more approval from Andrei

File: src/utils/calculations.ts [hunk 1]
```diff
@@ -215,3 +233,19 @@ export const getLoanCollateralDistribution = (loan: LoanResponseWithJson) => {
...
+  const pendingCollReward =
+    user.status !== 1 || isZero(diffCollRpt)
+      ? zeroCollateral()
+      : mulFrac(diffCollRpt, new Decimal(user.userStake.toString()), DECIMAL_FACTOR);
+
+  const diffStableRpt = market.stablecoinRewardPerToken.minus(user.userStablecoinRewardPerToken);
+  const pendingStableReward =
+    user.status !== 1 || diffStableRpt.isZero()
+      ? new Decimal(0)
+      : diffStableRpt.mul(user.userStake).div(DECIMAL_FACTOR);
```
Comment (andreihrs) [inline src/utils/calculations.ts:248]: did we test if the value shows up on par with what we see on chain? think we can check on devnet. I see that on the webapp we have the same calc, but in SC we have another division for precision rounding

**Review by andreihrs — APPROVED** [review]
lgtm

File: src/utils/calculations.ts (hunk 1 above)
Comment (peroxy) [reply src/utils/calculations.ts:248]: Good idea, I'll check it out locally for devnet and report the results shortly

**Pull request merged**